third_party/rust/BUILD

```diff
@@ -28,7 +28,6 @@ rust_bindgen_toolchain(
         ":specify_bindgen_libstdcxx": "@bindgen_libstdcxx_linux//:libstdc++",
         "//conditions:default": None,
     }),
-    system_includes = "@bindgen_clang_linux//:system_includes",
```

Comment:
Do you know what the purpose of this was in the first place?

Reply:
Yes, it was for adding the system toolchain's `include`s when installed in a different location. Since we patched this, `rules_rust` updated their bindgen toolchain to take the system includes from the hermetic C toolchain instead, so I don't think we need this anymore:

https://github.com/bazelbuild/rules_rust/commit/116f6db8fa6caa1f1cc7470d7f5a0d0e32b5352d

Reply:
@timothytrippel might be interested in how this affects the air gapped environment's setup.

Reply:
I think this should be ok, but we will have to see if this breaks the nightly regressions.

Reply:
Thanks, I think I'll break this PR up so we can revert the `rules_rust` update separately is regressions fail.